.rubocop.yml

```diff
@@ -108,7 +109,7 @@ Style/SpaceInsideHashLiteralBraces:
 Style/StringLiterals:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: double_quotes
+  EnforcedStyle: single_quotes
```

Comment:
Potentially the most controversial...  I can go either way, but the community standard (i.e. the rubocop default is for single quotes if you don't need interpolation) - and I prefer this way.